crates/pixi_config/src/lib.rs

```diff
@@ -696,6 +696,23 @@ impl From<&Config> for rattler_repodata_gateway::ChannelConfig {
 }
 
 impl Config {
+    /// Constructs a new config that is optimized to be used in tests.
+    ///
+    /// This instance is optimized to provide the fastest experience for tests.
+    pub fn for_tests() -> Self {
```

Comment:
could we use `#cfg[tests]` here?

Reply:
I want to expose this as a function from the crate. Also for other crates.

Reply:
got it now